src/index_UMD.ts

```diff
@@ -6,3 +6,19 @@ export * from './i18n';
 export * from './utils';
 export { getChannel } from './utils/getChannel';
 export * from './components/Emojis';
+
+declare global {
+  interface Window {
+    Channel: typeof import('stream-chat').Channel;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    ICAL: any;
+    logChatPromiseExecution: typeof import('stream-chat').logChatPromiseExecution;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    StreamChat: typeof import('stream-chat') & { StreamChat: any };
+  }
+}
+
+window.StreamChat.StreamChat = window.StreamChat;
+window.StreamChat.logChatPromiseExecution = window.logChatPromiseExecution;
+window.StreamChat.Channel = window.Channel;
+window.ICAL = window.ICAL || {};
```

Comment:
This used to be done in rollup. Since this fixup is only required for the browser bundle, it makes sense to just add it to bundle entrypoint. This file is not used anywhere else.

BTW, let me know if you have any idea what is ICAL.